### PR TITLE
Python SDK: Benchmarks

### PR DESCRIPTION
**Pull request by medikoo**
Few generalization steps, and setup of benchmarks and performance test for Python SDK.

Performance tast for Python is set with same threshold as one for Node.js and currently fails producing larger then expected latency for initialiation and first invocation.

Current benchmark results can be seen here: https://docs.google.com/spreadsheets/d/1UJWT6hkMv7YWMk1pCKmOjSl3GdNC76xCGOGs5XOyYVw/edit?usp=sharing (in other sheet for comparison Node.js results are provided)

See included in this PR documentation for more details

/cc @selcukcihan 

@czubocha it'll be nice to have it also setup in next step for Go SDK

### CODE REVIEW

File: node/test/python/aws-lambda-sdk/benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+###### `init:total`
+
+Total initialization time as reported by AWS (with `initDuration` metric provided in report of the first invocation)
+
+#### Invocation
+
+Benchmarks observe first invocation (`first`) (one that in regular circumstances immediately follows initialization), and the the second (`following`).
+
+Two first invocations are observed separately, as data shows that durations of first invocation are always worse than of the second, and that for the following (third, fourth etc.) invocations numbers are similar as for the second.
+_This difference could be attributed to how Node.js internal optimization works, where paths which were already taken usually run faster (it's to be confirmed whether we see similar differences with other runtimes)._
```
Comment (Danwakeem) [inline node/test/python/aws-lambda-sdk/benchmark/README.md]: This part talks about nodejs internal performance. Is that still true for this benchmark given it is written in node or should this be removed? 🤔

File: node/test/python/aws-lambda-sdk/benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+##### Observed metrics
+
+###### `[first|following]:internal:request`
+
+Invocation request handing duration in context of the Node.js internal extension. It's a time that's measured from the moment when our handler wrapper received an invocation event from AWS until we invoke the original Lambda handler.
```
Comment (Danwakeem) [inline node/test/python/aws-lambda-sdk/benchmark/README.md]: Same here referencing nodejs?

File: node/test/python/aws-lambda-sdk/benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+###### `[first|following]:internal:response`
+
+Invocation response handling duration in the context of the Node.js internal extension. It's a time that's measured from the moment when original handler provided response until we pass it back to AWS.
```
Comment (Danwakeem) [inline node/test/python/aws-lambda-sdk/benchmark/README.md]: Same here 🤔

**Review by Danwakeem — COMMENTED** [review]
This looks good. The benchmark doc is very helpful but I think might need a few small updates.

File: node/test/python/aws-lambda-sdk/benchmark/README.md (hunk 1 above)
Comment (medikoo) [reply node/test/python/aws-lambda-sdk/benchmark/README.md]: Good find, updated all cases 👍

**Review by Danwakeem — APPROVED** [review]
:shipit:

**Pull request merged**